fms/models/hf/utils.py

```diff
@@ -137,7 +137,8 @@ def to_hf_api(model: nn.Module, **override_config_kwargs) -> "HFModelArchitectur
 
 
 def as_fms_model(
-    model_id_or_path: Union[str, os.PathLike], device: Union[str, torch.device] = "cpu"
+    model_id_or_path: Union[str, os.PathLike],
+    **get_model_params,
```

Comment:
this seems unclear to me. `get_` makes it look like it's supposed to be a lambda, but it doesn't appear to be used that way.